components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -516,7 +516,7 @@ class GeckoEngineSession(
             screenY: Int,
             element: GeckoSession.ContentDelegate.ContextElement
         ) {
-            val hitResult = handleLongClick(element.srcUri, element.type, element.linkUri)
+            val hitResult = handleLongClick(element.srcUri, element.type, element.linkUri, element.title)
```

Comment:
Is this only available in 71.0 (`browser-engine-gecko-nightly`) for now?

Reply:
No, it was already  available in the others, missed it in here.

Reply:
Ah, okay :)